src/main/java/UseCase/createPlaylist.java

```diff
@@ -5,26 +5,41 @@
 import InputData.newPlaylistInputData;
 import Database.playlistAccessInterface;
 import Database.playlistLibrary;
+
+//import OutputData.newPlaylistOutputData;
+//import OutputBoundary.newPlaylistOutputBoundary;
+
 public class createPlaylist implements newPlaylistInputBoundary {
+    //TODO: Do I need to create an output data instance attribute ? I dont know if this is a data structure
```

Comment:
You technically don't need an outputData attribute, as you're creating it and then immediately passing it to the OutputBoundary/Presenter.

Reply:
Thats true, I think I made my code significantly more complicated by trying to use output data